### PR TITLE
MTDSA-27856 Added R15 updates for Sandbox release on 11/12/2024

### PR DESCRIPTION
**Pull request by AhmadZNaeem**

### CODE REVIEW

**Review by sion-docs — DISMISSED** [review]

**Review by MarkAKelly — CHANGES_REQUESTED** [review]
@Vedant-N421

File: README.md [hunk 1]
```diff
@@ -18,6 +18,281 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
...
+
+Added calculation type parameter `in-year` and `intent-to-finalise` for the following endpoint:
+- Trigger Tax Calculation
+- List Calculations
+
+Added calculation type parameter `intent-to-amend` for the tax year 2025-26 onwards for the following endpoint:
+- Trigger Tax Calculation
+- List Calculations
+
+Added new tax year specefic scehema for tax year 2023-24 onwards for the following endpoint:
```
Comment (Vedant-N421) [inline README.md:137]: "specific schema" instead of "specefic scehema"

File: README.md [hunk 2]
```diff
@@ -18,6 +18,281 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
...
+- Submit Foreign Property Accounting Adjustments
+
+Rename `typeOfBusiness` enum `uk-property-non-fhl` to `uk-property` for tax year before 2025-26 for the following endpoints:
+- Trigger a Business Source Adjustable Summary
+- List Business Source Adjustable Summaries
+
+Change enum values in request query parameters and response for field `typeOfBusiness` to `self-employment`, `foreign-property` and `uk-property` for tax year 2025-26 for the following endpoint:
+- List Business Source Adjustable Summaries
+
+Renamed enum value for field typeOfBusiness from uk-property-non-fhl to uk-property for tax year before 2025-26 for the following endpoint:
```
Comment (Vedant-N421) [inline README.md:192]: are uk-property-non-fhl (and uk-property) missing the backticks to make them "code-words"?

File: README.md [hunk 3]
```diff
@@ -18,6 +18,281 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
...
+API version 5.0 is now available in Sandbox, with the following change.
+
+##### Added
+
+Added error `RULE_BFL_NOT_SUPPORTED_FOR_FHL_PROPERTIES` and Gov-Test-Scenario `BFL_NOT_SUPPORTED_FOR_FHL_PROPERTIES` for the following endpoint:
+- Create a Brought Forward Loss
+
+Added error `RULE_CSFHL_CLAIM_NOT_SUPPORTED` and Gov-Test-Scenario `CSFHL_CLAIM_NOT_SUPPORTED` for tax year 2025-26 onwards for the following endpoints:
+- Create a Loss Claim
+  Amend a Loss Claim Type
```
Comment (Vedant-N421) [inline README.md:256]: Missing the '-' here I think

**Review by Vedant-N421 — APPROVED** [review]

**Pull request merged**